.github/dependabot.yml

```diff
@@ -0,0 +1,11 @@
+version: 2
+updates:
+# Maintain dependencies for GitHub Actions
+- package-ecosystem: github-actions
+  directory: /
+  schedule:
+    interval: monthly
+  groups:
+    gha-dependencies:
+      patterns:
+      - '*'
```

Comment:
Will this not trigger if there are no actual updates? I can't be arsed having to deal with automated PRs that don't actually do anything.

Reply:
No superfluous PRs, guaranteed, I promise, really!
 (I've had this config on a couple repos for a while and no problems). 

Note also that in this config, if there are multiple different actions that need update, only one PR is created that combines them all. So you'll need to do at most one click per month. :stuck_out_tongue: